src/app/(components)/Navbar.css

```diff
@@ -0,0 +1,16 @@
+.navbarContainer {
+    background-color: red;
```

Comment:
The red background was a visual help to make you understand the size of the navbar, feel free to change it to something more realistic if you want.

Reply:
will do